projects/kit/src/lib/masks/number/processors/thousand-separator-postprocessor.ts

```diff
@@ -22,21 +22,21 @@ export function createThousandSeparatorPostprocessor({
         return identity;
     }
 
-    const prefixReg = new RegExp(`^${escapeRegExp(prefix)}${CHAR_MINUS}?`);
-    const postfixReg = new RegExp(`${escapeRegExp(postfix)}$`);
     const isAllSpaces = (...chars: string[]): boolean => chars.every(x => /\s/.test(x));
 
     return ({value, selection}) => {
-        const [integerPart, decimalPart = ''] = value.split(decimalSeparator);
+        const {cleanValue, extractedPostfix, extractedPrefix} = extractPrefixAndPostfix({
```

Comment:
now we firstly removing prefix and postfix from value, and then splitting cleanValue to numberParts